.travis.yml

```diff
@@ -12,15 +12,30 @@ node_js:
   - "11"
 
 install:
-  - npm ci
-  - npx lerna bootstrap --ci
+  - npm install
+  - npx lerna bootstrap --no-ci
 
 script:
   - npm run test
   - npm run lint
   - node scripts/check-dependencies.js
 
-cache: npm
+cache:
```

Comment:
I had to change so many things that I also added this. I'm not convinced that this makes the builds faster.